crates/apub/src/protocol/objects/group.rs

```diff
@@ -98,3 +99,27 @@ impl Group {
     Ok(())
   }
 }
+
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+#[serde(untagged)]
+pub(crate) enum AttributedTo {
+  Lemmy(CollectionId<ApubCommunityModerators>),
+  Peertube(Vec<AttributedToPeertube>),
+}
```

Comment:
For this you can also use the `AttributedTo` from page.rs, by moving it into mod.rs and using vec instead of fixed length array.